main.go

```diff
@@ -397,7 +400,7 @@ func main() {
 		CollectSystemInformation: !noSystemInformation,
 		StateFilename:            stateFilename,
 		WriteStateUpdate:         (!dryRun && !dryRunLogs && !testRun) || forceStateUpdate,
-		ForceEmptyGrant:          dryRun || dryRunLogs || benchmark,
+		ForceEmptyGrant:          dryRun || dryRunLogs || testRunLogs || benchmark,
```

Comment:
This was needed for the test suite to pass. It's not clear from the `main.go` documentation what the difference is between `--test-logs` and `--dry-run-logs`. Maybe one of them should be dropped?

Reply:
Hmm, so I think the different is:

* `--test-logs` is supposed to get a log grant, and then either do a log download or log tail
* `--dry-run-logs` is supposed to work without an API server, so skips the grant

However, I'm not sure if the distinction matters.

Really the main use case is `--test-logs`, which we document, to verify whether the log setup is correct. I suppose doing a log grant could be beneficial for Enterprise environments (since I think it would trigger the server side to check for correct LOG_ENCRYPTION_KEY / bucket setup?). Though that's soon to be removed anyway.

Reply:
I'm not sure what the right course of action is here. Is there a blocker?

Reply:
FWIW, I also don't think this is a blocker. I haven't seen much benefit of `--dry-run-logs`, so I'm good with dropping it.

That said, the regular `--dry-run` is useful though (it helps get clarity on what data the collector sent), so we shouldn't be messing with that, I think :)

Reply:
Sorry, I don't think I explained my reasoning here well. I'm thinking that there's little benefit to actually submitting test snapshots to the API, so in dropping `--dry-run-logs` my intention was to make `--test-logs` a dry run command. Then we only need to contact the API for a top-level `--test`.